gui/src/installer/step/node/electrum.rs

```diff
@@ -0,0 +1,73 @@
+use iced::Command;
+use liana::{
+    config::ElectrumConfig,
+    electrum_client::{self, ElectrumApi},
+};
+use liana_ui::{component::form, widget::*};
+
+use crate::{
+    installer::{
+        context::Context,
+        message::{self, Message},
+        view, Error,
+    },
+    node::electrum::ConfigField,
+};
+
+#[derive(Clone)]
+pub struct DefineElectrum {
+    address: form::Value<String>,
+}
+
+impl DefineElectrum {
+    pub fn new() -> Self {
+        Self {
+            address: form::Value::default(),
+        }
+    }
+
+    pub fn can_try_ping(&self) -> bool {
+        !self.address.value.is_empty() && self.address.valid
+    }
+
+    pub fn load_context(&mut self, _ctx: &Context) {}
```

Comment:
I think it can be removed. It's a leftover from an earlier version where I defined a trait that each node type had to implement, but now it's not required.

Reply:
(for future reference: I removed the trait as I was having trouble including the `ping()` method in an object-safe way. I only wanted the ping method to return the `Result` and not the `Command` to handle the `PingResult`)